src/ColumnSelectionStep/ColumnSelectionStep.tsx

```diff
@@ -123,6 +124,35 @@ function GeneralizationControls({
   }
 }
 
+function NullsInAidWarning(schema: TableSchema, aidColumn: string) {
+  const computedResult = useNullAid(schema, aidColumn);
+
+  switch (computedResult.state) {
+    case 'completed': {
+      return computedResult.value ? (
+        <>
+          <Alert
+            className="ColumnSelectionStep-notice"
+            message={
+              <>
+                <strong>CAUTION:</strong>The protected entity identifier column contains NULL or empty values.
+              </>
+            }
+            type="warning"
+            showIcon
+            icon={<InfoCircleOutlined />}
+            closable
+          />
+        </>
+      ) : null;
+    }
+
+    case 'in_progress':
+    case 'failed':
+      return null;
```

Comment:
and a generic error here, like "Something went wrong" for example (as it should almost never happen anyway).

Reply:
again - where should the error show up and what should it say, without freaking the user out unnecessarily? Given the circumstances and the secondary nature of this warning, I'd vote to not show anything, since 99,9% of the time the Aid column is fine.

Reply:
I guess it's ok without it, since this check is optional anyway.